docs/vercel.json

```diff
@@ -480,6 +480,31 @@
       "destination": "/etl-pipeline-tutorial/",
       "permanent": false
     },
+    {
+      "source": "/tutorials",
+      "destination": "/examples",
```

Comment:
To redirect everything that was in `/tutorials/...` to `/examples/...` you will want to do:

```
    {
      "source": "/tutorials/:path*",
      "destination": "/examples/:path*",
      "permanent": false
    },
    ```